### ISSUE TITLE
China, PLAN, Type 054A, all batch Combat System Generation modification

### ISSUE THREAD

**Issue by LuigiP57** [PygmalionOfCyprus/cmo-db-requests#4858]
### DB Selector

DB3K

### Affected DBID(s)

Ship 1965 2714 2495 4582 4723 4724 4725

### Summary of Changes

- [ ] Change CS Gen to 6

I didn't bring any new "evidence" this time, but simply cross-referenced the database. I found out: the Indian Navy's Talwar and Shivalik class frigates, both of which are older than the 054a first ship but have Gen 6 2000-2029 class CS. The U.S. Navy Burke class is also the CS Gen 6, starting with the DDG-91, which entered service in 2004, also before the first ship 054A entered service.

​Before building the 054A, the PLAN already had the Type 052B, a modern air defense ship, so it made perfect sense for the 054A to have the 21st century CS.

### Sources

![image](https://github.com/user-attachments/assets/b32e8cd4-8243-4ade-98c4-d4e6c1da1f72)
![image](https://github.com/user-attachments/assets/6773e166-3e39-4e8d-a35c-73e7584c9f96)
![image](https://github.com/user-attachments/assets/e1f18581-94c6-4082-8a14-423a72da771a)

**Comment by HunterDNiu** [comment]
Could have been from 054 which is from Lafayette class... Should be fixed tho

**Comment by LuigiP57** [comment]
> Could have been from 054 which is from Lafayette class... Should be fixed tho

I will only give my personal opinion: the Type 054 frigate is only two built, and is the first modern frigate of the PLAN, and the technology they use on it **may not be as mature as the Type 054A**. As such, I am hesitant to put too optimistic expectations on the "then" PLAN. However, it is common for ships to be iteratively upgraded from the baseline version, and the Type 054A can be regarded as an upgraded version of the Type 054, and has a even-modernized air defense system, so we think it is reasonable to improve its CS. :)

**Comment by LuigiP57** [comment]
I am sure you are also aware of a widely known "open secret" about the PLAN (or the PLA as a whole) : they are catching up with advanced technology by rapidly building a small but diverse range of equipment. For example, the Type 093 SSNs, none of them are the same as each other.

**Comment by PygmalionOfCyprus** [comment]
I would not put much stock into cross-referencing non-China units that have not been reviewed in the DB. The Pakistani/Indian frigates may have a higher CS Gen, but this is probably not accurate. When we initially did the CS/OODA review, CS generations were assigned purely based on in-service date; by contrasts, when I did the PLAN review, I specifically went through and took note of individual combat systems and tried to estimate their capabilities. 

You will note that the 052B -- indeed a modern air-defense ship -- also has a Gen 5 CS, though its ergonomics were upgraded to "Good" during its MLU (when they replaced a lot of the old terminals with newer, nicer ones but did not as far as I know actually completely replace the CS software/hardware).

The 052s and 054As have the same combat system AFAIK: the H/ZKJ-5, which while better than the ancient systems they were using before was still very much a "learning experience" for the PLAN. Good enough for mass deployment, but not good enough to achieve parity vs. Western systems of the same time period. That's what I tried to represent w/ Gen 5 CS. You'll note of course that I did up the ergonomics on these ships as they went through MLUs, Batch updates, etc. as with the 052B.

I believe China's newer ships are up to, like...the ZKJ-7, or something? Of course I don't know for sure the capabilities of these new systems but I feel much more confident marking those ships as Gen 6, which I have. The system may be imperfect but I feel it fairly accurately represents the Chinese capabilities in comparison to one another right now. If we start treating ZKJ-5 era stuff as "Gen 6" then I will need to start marking the newest vessels as Gen 6 / Excellent Ergo or even Gen 7, which would make them the best units in the game -- and I think even the Chinese would not argue their CS systems have reached that level yet!

I have knocked down the Tughril class CS gen to match the Chinese counterparts, though.
